README.md

```diff
@@ -0,0 +1,32 @@
+## How to Make Changes to the Group Website
+
+### A. From your own forked repo
+
+#### I. Getting Started
+
+1. Fork the original [repo](https://github.com/az-digitalag/group-website)
```

Comment:
* If in the org, option to just create a branch?

Reply:
Yes I was going to include that as well

Reply:
I think most of these instructions should be the same whether it's from a branch off the main repo, a fork of the main repo, or a branch off a fork. So I'll make some suggestions for how to handle that.

Reply:
```suggestion
1. Either clone the original [repo](https://github.com/az-digitalag/group-website) to your local computer and always work off of a non-master branch, or fork the original repo and clone the forked repo to your computer. You can additionally work off of a branch on the forked repo if you'll be working on multiple different tasks at a time. 
```